newstyle.css

```diff
@@ -272,3 +272,88 @@ a:hover {
     color: #84c754;
 }
 </style>
+<style>/* Stylesheet 4: */
+body {
+    font: 100% Courier New;
+    margin: 20px;
+    line-height: 26px;
+    background-color: #000000;
+}
+
+.container {
+    xmin-width: 900px;
```

Comment:
xmin-width should be 800px instead.

Reply:
got it, check again

Reply:
i think it is good, and I am going to commit it now.